Hw_02/Homework2.rmd

```diff
@@ -0,0 +1,157 @@
+---
+title: "HW02"
+author: "Tyler Allan"
+date: "2/5/2021"
+output: html_document: default
+        github_document: default
+---
+```{r}
+library(gapminder)
+library(tidyverse)
+library(ggplot2)
+library(dplyr)
+```
+
+#1.1 Use filter() to subset the gapminder data to three countries of your choice in the 1970's
+```{r}
+acjminder <- gapminder %>% 
+  filter(country %in% c("Afghanistan","Canada","Japan") & year > 1969 & year < 1980)
+print(acjminder)
+```
+
+
+#1.2 Use the pipe operator to select "country" and "gdpPercap" from your filtered dataset in 1.1
+``` {r}
+acjminder %>% 
+  select(country, gdpPercap)
+```
+
+#1.3 Make a new variable in gapminder for the change in life expectancy from the previous measurement. Filter this table to show all of the entries that have experienced a drop in life expectancy. Hint: you might find the lag() and diff() functions useful.
+```{r}
+deltalifeExp <- gapminder %>% 
+  mutate(deltalifeExp = lag(lifeExp, order_by = year))
+arrange(deltalifeExp, year)
+deltalifeExp %>% 
+  filter(deltalifeExp < lifeExp)
+```
+
+#1.4 Filter gapminder so that it shows the max GDP per capita experienced by each country. Hint: you might find the max() function useful here.
+```{r}
+gapminder %>% 
+  group_by(country) %>% 
+  filter(gdpPercap %in% max(gdpPercap))
```

Comment:
Is "%in% serving a similar function as "==" if so, this is new to me!

Reply:
%in% makes it so the countries can share values in the subset. When I was using == the output only allowed one country to be associated with a particular year.